okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -342,14 +361,19 @@ private boolean shouldRetry(IOException exception, Optional<Duration> backoff) {
     private void scheduleExecution(
             Duration backoff,
             Tags.AttemptSpan attemptSpan,
-            Runnable execution) {
+            Consumer<Callback> enqueueFunction,
+            Callback callback) {
         DetachedSpan backoffSpan = attemptSpan.attemptSpan().childDetachedSpan("OkHttp: backoff-with-jitter");
 
         // TODO(rfink): Investigate whether ignoring the ScheduledFuture is safe, #629.
         schedulingExecutor.schedule(
-                () -> executionExecutor.submit(() -> {
+                () -> executionExecutor.execute(() -> {
```

Comment:
if the provided runnable throws, execute will allow the uncaught exception handler to log it, while submit assumes the returned future is read (but we ignore it)